tools/wptrunner/wptrunner/browsers/epiphany.py

```diff
@@ -24,9 +24,14 @@ def check_args(**kwargs):
 
 
 def browser_kwargs(test_type, run_info_data, config, **kwargs):
+    webdriver_args = kwargs.get("webdriver_args")
+    # Workaround for https://gitlab.gnome.org/GNOME/libsoup/issues/172
+    if not any('--host=' in item for item in webdriver_args):
```

Comment:
```suggestion
    if not any(item.startswith('--host=') for item in webdriver_args):
```

A bit easier to read with way I think, and also avoids weird cases where "--host=" is in the argument but not at the beginning.

Reply:
Indeed, good suggestion :)